packages/donor/src/components/Wizard/WizardScreen.module.scss

```diff
@@ -1,30 +1,58 @@
 .container {
+  height: 100%;
```

Comment:
for some reason now on devtools in the smaller devices the next button is out of the screen and i can't scroll down to it

Reply:
I will fix it in a different commit